fe-nc-news/src/Comments.jsx

```diff
@@ -0,0 +1,35 @@
+import getComments from './utils/GetComments'
+import {React, useState, useEffect} from 'react'
+
+const Comments = ({article_id}) => {
+ const [comments, setComments] = useState([])
```

Comment:
It might be helpful for the user to have an indication of whether comments are loading. 
Every time we're making an asynchronous request, there could be a delay in receiving our data.